### ISSUE TITLE
Mock code challenge has no tests

### ISSUE THREAD

**Issue by Pete-Fowler** [learn-co-curriculum/Active-Record-Theater-Work#7]
### Canvas Link

https://learning.flatironschool.com/courses/5222/assignments/208148?module_item_id=479909

### Concern

This has no tests, so the student probably has no idea if their code works. This is also inconsistent with the rest of the curriculum, where all mock CC's have tests.

### Suggested Changes

Add tests.

**Comment by arbarrington** [comment]
I agree. Pre-written seed files would be very helpful if tests are not possible.

**Comment by dalytimdaly** [comment]
agreed. the tests really help affirm knowledge and help with problem-solving. it's unfortunate that the most important part of the phase lacks this aspect.
